src/Nethermind/Nethermind.Core/Transaction.cs

```diff
@@ -57,6 +58,13 @@ public class Transaction
         public UInt256 Timestamp { get; set; }
 
         public AccessList? AccessList { get; set; } // eip2930
+        public UInt256? MaxFeePerDataGas { get; set; } // eip4844
```

Comment:
What about changes in TxDecoder? separate PR?

Reply:
Yes, all the encoding part will be a separate RP